src/metabase/db/metadata_queries.clj

```diff
@@ -126,7 +166,11 @@
                                                  [:expression expression-name]
                                                  [:field (u/the-id field) nil])))
                           :limit        limit}
-                   order-by (assoc :order-by order-by))
+                   order-by
+                   (assoc :order-by order-by)
+
+                   (:database_require_filter table)
```

Comment:
table-sample-rows for bigquery will use this query when sync a time ingestion table